### DISCUSSION TITLE
Date and Time components

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#2818] [General] [answered]
Would love to see some date and time input components.

**Comment by chr-ge** [comment, marked as answer]
https://github.com/chakra-ui/chakra-ui/issues/2612
> Building a datepicker that meets our accessibility standards while covering the very long list of use cases is a massive undertaking that we simply don't have the time or energy to implement or support. Hope you understand!Building a datepicker that meets our accessibility standards while covering the very long list of use cases is a massive undertaking that we simply don't have the time or energy to implement or support. Hope you understand!

**Reply by ghost** [reply]
Oh, of course! It certainly wasn't a demand or a complaint. Just a suggestion. :-)

I love what you've got, already.
